backend/report.js

```diff
@@ -293,6 +304,8 @@ function createProjection(filterObj) {
     };
     if(filterObj.isCommentNeeded) {
```

Comment:
This condition is used only if we need  make aggregation for comment field. 
This parameter igoes from client side. 
Please use different condition. For example add the similar flag. Just search by "isCommentNeeded" string to know how to use it.